.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -12,4 +12,5 @@
 <!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
 - [ ] I have followed the [contribution guidelines](../CONTRIBUTING.md) and code style for this project.
 - [ ] I have added tests covering my contributions.
-- [ ] I have updated the documentation accordingly.
\ No newline at end of file
+- [ ] I have updated the documentation accordingly.
+- [ ] This PR is being made to `staging` and not `master`.
```

Comment:
This is particularly confusing for me, and I have previously made PRs into Master, which were also mistaken accepted.

It would make things much easier if the "Default Branch" in the repo branch "Settings" was set to staging instead of "Master" to prevent this from happening with people who are expected master to be the main branch.

![image](https://user-images.githubusercontent.com/9027725/70082384-4bf9c200-15d8-11ea-975a-eb309a420417.png)

Reply:
the idea is to have staging as developing branch while master is our stable branch, most users will just download master, where (hopefully) the code works always. In staging code can fail